package.json

```diff
@@ -47,11 +47,11 @@
     "react-dom": "^17.0.2",
     "regenerator-runtime": "^0.13.7",
     "resource-workspace-rcl": "2.1.0",
-    "scripture-resources-rcl": "5.2.2",
-    "single-scripture-rcl": "3.2.0",
+    "scripture-resources-rcl": "5.3.9-beta",
```

Comment:
I see there is a recent release of `scripture-resources-rcl` but I don't see any PR in that repo.